demo/kubernetes/rook-client/rook-client.yml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: rook-client
+spec:
+  containers:
+  - name: rook-client
+    image: quay.io/rook/k8s-demo
+    imagePullPolicy: IfNotPresent
+    command: ["sleep", "36500d"]
```

Comment:
maybe we could roll this into the image (i.e. make this command the default command) or better yet, use the pause binary from gcr.io/google_containers/pause which hangs until kill signal (instead of sleeping for a very long time)